components/ui/Image.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.